history-network.md

```diff
@@ -206,9 +206,11 @@ ssz_uncles              = SSZList(encoded_uncles: Byte, max_length=MAX_ENCODED_U
 encoded_uncles          = rlp.encode(list_of_uncle_headers)
 ```
 
-Note the type-specific encoding might be different in future transaction types, but this encoding
+Note 1: The type-specific encoding might be different in future transaction types, but this encoding
 works for all current transaction types.
 
+Note 2: The `list_of_uncle_headers` refers to the array of uncle headers [defined in the devp2p spec](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#block-encoding-and-validity).  
```

Comment:
Using devp2p as the reference is not wrong, but also not helpful, maybe? It's really the yellow paper spec that defines it. That format is consensus-relevant, because the uncles hash found in the block header is the hash of `rlp.encode(list_of_uncle_headers)`. That's the primary reason for the choice. I would have a hard time using "devp2p did it this way" as a primary justification for any of our design choices :P

Reply:
The attempt to describe the rationale is here: https://github.com/ethereum/portal-network-specs/blob/e807eb09d2859016e25b976f082735d3aceceb8e/history-network.md#encoding-content-values-for-validation

Reply:
Ah, gotcha, sorry.  I remember that in your PR and I'm good with it.  I just wanted to explain what `list_of_uncle_headers` actually was since it's not clearly defined that it's an array of byte arrays where each byte array is the raw bytes representation of the uncle header.

Reply:
I revised my reference to point to the yellow paper and then further define in hopefully clear language how the uncles should be represented.

Reply:
Cool looks good, thanks for adding!